src/utility/HCISharedMemTransport.cpp

```diff
@@ -553,13 +553,35 @@ size_t HCISharedMemTransportClass::write(const uint8_t *data, size_t length)
 //private:
 void HCISharedMemTransportClass::start_ble_rf(void)
 {
-  if ((LL_RCC_IsActiveFlag_PINRST()) && (!LL_RCC_IsActiveFlag_SFTRST())) {
-    /* Simulate power off reset */
-    LL_PWR_EnableBkUpAccess();
-    LL_PWR_EnableBkUpAccess();
-    LL_RCC_ForceBackupDomainReset();
-    LL_RCC_ReleaseBackupDomainReset();
-  }
+	/* HW semaphore Clock is enabled for this variant */
+	LL_AHB3_GRP1_EnableClock(LL_AHB3_GRP1_PERIPH_HSEM);
+
+	/* This prevents the CPU2 (M0+) to configure RCC */
+	while (LL_HSEM_1StepLock(HSEM, CFG_HW_RCC_SEMID));
+
+#ifdef LL_APB1_GRP1_PERIPH_PWR
```

Comment:
As I understand this file is specific to STM32WB, 
and LL_APB1_GRP1_PERIPH_PWR is Not defined for WB.

Reply:
True, it can be removed, but (do not forget to) restore in case another device also use this HCI SharedMemTransport

Reply:
Removed.